### PR TITLE
Styling bookstore

### PR DESCRIPTION
**Pull request by Felipe-Perez-Ferraro**
- Added stylings to the proyect
- modified BooCard.jsx
- Modified the links
- Added the select input
- Updated BookList.jsx
- Updated Categories.jsx

### CODE REVIEW

File: src/App.jsx [hunk 1]
```diff
@@ -28,17 +28,17 @@ function App() {
...
                 className={
-                  activeLink === 'home' ? 'activeLink' : 'header__link__item'
+                  activeLink === 'books' ? 'activeLink' : 'header__link__item'
                 }
-                onClick={() => handleClick('home')}
+                onClick={() => handleClick('books')}
               >
-                Home
+                Books
               </Link>
```
Comment (Strangeal) [inline src/App.jsx]: - [ ] Great job styling your bookstore web app so beautifully 🥰. However, I noticed you missed little details of the design that when added your design is going to look perfect 👌. Kindly consider the below suggestions to help you add these missing details:

 - Kindly consider adding the user `profile`  icon at the far end of your `nav` as shown on the original design given in the project requirements. See below for more details:
 
<table>
<tr>
<th align=“middle”>Original design</th>
<th align=“middle”>Your design</th>
</tr>
<tr>
<td>
Screenshot 2023-08-24 at 17 23 12
</td>
<td>
Screenshot 2023-08-24 at 17 22 45
</td>
</tr>
</table>

File: src/components/bookList/BookList.css [hunk 2]
```diff
@@ -1,52 +1,106 @@
...
-  flex-wrap: wrap;
+  justify-content: space-evenly;
+  margin-top: 20px;
+  margin-bottom: 20px;
   gap: 10px;
+  width: 100%;
+  background-color: var(--white);
+  padding: 8px;
+  border: 1px solid var(--border);
 }
```
Comment (Strangeal) [inline src/components/bookList/BookList.css]: - [ ] Great job so far styling your cards. However, looking at the original design given in the project requirements carefully you would notice that the `padding` you have at the `top` and `bottom` of your is a bit smaller as compared to the original design so kindly add some for padding to it to match that of the design given.

- [ ] Kindly consider increasing your `max-width` as it's a bit smaller compared to that of the original design. I would suggest you use `80vw` as that might help but yes play around with the width and know what best matches the original design. See below for more details:

- [ ] Also, Your design is missing the `progress` bar on the card. Kindly consider adding it so your design matches that of the original.
<table>
<tr>
<th align=“middle”>Original design</th>
<th align=“middle”>Your design</th>
</tr>
<tr>
<td>
Screenshot 2023-08-24 at 17 14 24
</td>
<td>
Screenshot 2023-08-24 at 17 12 03
</td>
</tr>
</table>

**Review by Strangeal — CHANGES_REQUESTED** [review]
Hi @Felipe-Perez-Ferraro, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!



## Highlights
- Good **PR** title ✅ 
- Descriptive *PR* description ✅ 
- Implement the features ✅ 
- Nice *UI* design ✅  
- Professional `README` template ✅ 
- Correct `GitHub` workflow ✅ 
- Meaningful **Github** commits ✅ 
- Correct `.github/workflows` for linters ✅ 
- Followed *best-practices* ✅ 
- Correct **Semantics** used ✅ 
- Well structured project ✅ 

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by ademibowale — APPROVED** [review]
Hi @Felipe-Perez-Ferraro 


Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## To Highlight 💯 
- No linters errors ✔️
- styling exactly like original given design ✔️
- Added stylings to Bookstore app✔️
- Descriptive pull request ✔️ 
- Descriptive commit messages ✔️ 
- Good use of Github flow ✔️






- [x]  _Please check the comments under the review._

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me on( @Adebowale ) in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**